tnb/validators.py

```diff
@@ -29,7 +29,7 @@ def fetch_account_balance_lock(self, account_number):
         """
         return self.fetch(f"/accounts/{account_number}/balance_lock")
 
-    def fetch_confirmation_block(self, block_identifier):
+    def fetch_confirmation_block(self, block_identifier: str) -> object:
```

Comment:
This method returns a `dict`.

**Note**: The method documentation is also incorrect. 

```suggestion
    def fetch_confirmation_block(self, block_identifier: str) -> dict:
```